tests/integration/test_outcome.py

```diff
@@ -593,9 +593,9 @@ def _get_event_payload(event_type):
             },
             "transaction": "hi",
         }
-    elif event_type == "feedback":
+    elif data_category == "user_report_v2":
         return {
-            "type": "feedback",
+            "type": "userreportv2",
```

Comment:
This is the name of the event item type, because of https://github.com/getsentry/relay/blob/0608f08570724e8e6b6667e68c382af643f14fd9/relay-base-schema/src/events.rs#L30

Reply:
According to the coverage in `test_feedback.py` this is the correct type.

https://github.com/getsentry/relay/blob/master/tests/integration/test_feedback.py#L6